applications/video_streaming/video_streaming_client/python/streaming_client_demo.py

```
@@ -1,3 +1,4 @@
+#!/usr/bin/env python3
```

Ruff (0.14.1)

Shebang is present but file is not executable

(EXE001)